packages/expo-in-app-purchases/src/InAppPurchases.types.ts

```diff
@@ -0,0 +1,29 @@
+export type ValidItemType = 'inapp' | 'subs';
+
+export interface QueryResponse {
+  responseCode: number,
+  results: Array<Purchase | ItemDetails>,
+}
+
+export interface Purchase {
+  acknowledged: boolean,
+  orderId: string,
+  productId: string,
+  purchaseState: number,
+  purchaseTime: number,
+  packageName: string, // Android only
+  purchaseToken?: string // Android only
+  transactionReceipt?: string // iOS only
+}
+
+export interface ItemDetails {
+  description: string,
+  price: string,
+  price_amount_micros: number,
```

Comment:
I agree camel case is preferred. I originally named it that because that's what [Android returns](https://developer.android.com/reference/com/android/billingclient/api/SkuDetails.html#getpriceamountmicros) and I was calling `skuDetails.getOriginalJson()` which returned the price amount micros field using underscore case, however I can change that natively now that I no longer do that.